AsyncDisplayKit/TextKit/ASTextKitFontSizeAdjuster.mm

```diff
@@ -85,7 +87,7 @@ - (NSUInteger)lineCountForString:(NSAttributedString *)attributedString
   NSLayoutManager *layoutManager = _attributes.layoutManagerCreationBlock ? _attributes.layoutManagerCreationBlock() : [[ASLayoutManager alloc] init];
   layoutManager.usesFontLeading = NO;
   [textStorage addLayoutManager:layoutManager];
-  NSTextContainer *textContainer = [[NSTextContainer alloc] initWithSize:_constrainedSize];
+  NSTextContainer *textContainer = [[NSTextContainer alloc] initWithSize:CGSizeMake(_constrainedSize.width, FLT_MAX)];
```

Comment:
Please consider adding a comment here in your next diff, basically that this code is only used for the purposes of testing whether we fit / learning the true line count so we don't need to limit it.

One question / suggestion: it might actually be faster to set a limit here that is, say, one line greater than what we actually want the string to fit in.  This would allow the layout operation to start earlier if we're trying it at a size where it clearly isn't going to fit.  E.g. if you want something to fit in 3 lines but it's actually 10 at a large starting font size, we should be able to know by line 4 that it isn't going to fit.